sky/backends/backend_utils.py

```diff
@@ -139,9 +139,10 @@
     ('available_node_types', 'ray.worker.default', 'node_config', 'UserData'),
 ]
 
-_SET_RAY_ADDRESS = ('RAY_PORT=$(python -c "from sky.skylet import job_lib; '
-                    'print(job_lib.get_ray_port())" 2> /dev/null || echo 6379);'
-                    'RAY_ADDRESS=127.0.0.1:$RAY_PORT ')
+RAY_STATUS_WITH_SKY_RAY_PORT_CMD = (
+    'RAY_PORT=$(python -c "from sky.skylet import job_lib; '
+    'print(job_lib.get_ray_port())" 2> /dev/null || echo 6379);'
```

Comment:
Does this extra python invoke add latency to `sky launch` or `status -r`?

Reply:
The extra python invoke adds about 0.75s overhead to the original ray status call. (1.25 -> 2.0) by comparing the following two commands:
```
time ray status
time { RAY_PORT=$(python -u -c  "from sky.skylet import job_lib; print(job_lib.get_ray_port())" 2> /dev/null || echo 6379);RAY_ADDRESS=127.0.0.1:$RAY_PORT ray status; }
```

Reply:
Seems a lot standalone, but considering `sky launch` or `status -r` take minutes and tens of seconds, 0.75s *x seems ok?

Reply:
Yes, it might not be that significant compared to the total time. I am going to merge this for now, and we can think of optimization later, if needed. : )